apps/web/tests/asset-inventory/collector/collector-details/edit-collector.spec.ts

```diff
@@ -0,0 +1,295 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Edit Schedule', () => {
+    let page;
+
+    /* Init */
+    test.beforeEach(async ({ browser }) => {
+        page = await browser.newPage();
+    });
+
+    test.afterEach(async () => {
+        await page.close();
+    });
+
+    // Make random string for key/value.
+    const random = () => Math.random();
+    const randomKey = `Key-${random()}`;
+    const randomValue = `Value-${random()}`;
+
+    const ORIGIN_COLLECTOR_NAME = 'for_testcode';
+    const NEW_COLLECTOR_NAME = 'for_testEdit';
+    let currentCollectorName: '';
+
+    const goToCollectorPage = async () => {
+        await page.goto('/asset-inventory/collector');
+        const pageTitleName = page.locator('.heading-wrapper>h2.has-right>span.title');
+
+        await expect(pageTitleName).toContainText('Collector');
+    };
+    const saveChanges = async (index: number) => {
+        const saveButton = page.locator('.collector-detail-page .section').nth(index).locator('.save-changes-button');
+        await saveButton.first().click();
+    };
+
+    /* Test */
+    test('Edit collector schedule', async () => {
+        await test.step('1. Go to collector page', async () => {
+            await goToCollectorPage();
+        });
+
+        await test.step('2. Click on the first collector and go to the details page', async () => {
+            // Click collector.
+            const collector = await page.locator(`.collector-lists>div .collector-item-name:text('${ORIGIN_COLLECTOR_NAME || NEW_COLLECTOR_NAME}')`);
```

Comment:
In the development environment, as multiple people are testing, there were cases where the content overlapped, and since the testing environment varies each time, I made it explicit!

Reply:
I got it! then what about to use just COLLECTOR_NAME_1, COLLECTOR_NAME_2?